reference/supervisor.md

```diff
@@ -383,18 +384,28 @@ const char *wb_supervisor_node_get_base_type_name(WbNodeRef node);
 namespace webots {
   class Node {
     enum {
-      NO_NODE, ACCELEROMETER, APPEARANCE, BACKGROUND, BALL_JOINT, BALL_JOINT_PARAMETERS,
-      BOX, BRAKE, CAMERA, CAPSULE, CHARGER, COLOR, COMPASS, CONE, CONNECTOR, CONTACT_PROPERTIES,
-      COORDINATE, CYLINDER, DAMPING, DIFFERENTIAL_WHEELS, DIRECTIONAL_LIGHT, DISPLAY,
-      DISTANCE_SENSOR, ELEVATION_GRID, EMITTER, EXTRUSION, FOCUS, FLUID, FOG, GPS,
-      GROUP, GYRO, HINGE_2_JOINT, HINGE_2_JOINT_PARAMETERS, HINGE_JOINT, HINGE_JOINT_PARAMETERS,
-      IMAGE_TEXTURE, IMMERSION_PROPERTIES, INDEXED_FACE_SET, INDEXED_LINE_SET, INERTIAL_UNIT,
-      JOINT_PARAMETERS, LED, LENS_DISTORTION, LIDAR, LIGHT_SENSOR, LINEAR_MOTOR,
-      MATERIAL, MICROPHONE, PEN, PHYSICS, PLANE, POINT_LIGHT, POSITION_SENSOR, PROPELLER,
-      RADAR, RADIO, RANGE_FINDER, RECEIVER, RECOGNITION, ROBOT, ROTATIONAL_MOTOR,
-      SHAPE, SLIDER_JOINT, SLOT, SOLID, SOLID_REFERENCE, SPEAKER, SPHERE, SPOT_LIGHT,
-      SUPERVISOR, SWITCH, TEXTURE_COORDINATE, TEXTURE_TRANSFORM, TOUCH_SENSOR, TRACK,
-      TRACK_WHEEL, TRANSFORM, VIEWPOINT, WORLD_INFO, ZOOM
+      NO_NODE,
+      // VRML97 nodes
+      APPEARANCE, BACKGROUND, BOX, COLOR, COMPOSED_CUBE_MAP_TEXTURE, COMPOSED_SHADER,
+      CONE, COORDINATE, CYLINDER, DIRECTIONAL_LIGHT, ELEVATION_GRID, EXTRUSION,
+      FOG, GROUP, IMAGE_TEXTURE, INDEXED_FACE_SET, INDEXED_LINE_SET, LIDAR, MATERIAL,
+      MULTI_TEXTURE, MUSCLE, POINT_LIGHT, SHADER_PART, SHAPE, SPHERE, SPOT_LIGHT,
+      SWITCH, TEXTURE_COORDINATE, TEXTURE_TRANSFORM, TRACK, TRACK_WHEEL, TRANSFORM,
+      UNIFORM, VIEWPOINT, WORLD_INFO,
+      // non-vrml geometry
```

Comment:
We should spell it `non-VRML97 geometry nodes` to be consistent with previous comment.